### PR TITLE
Total Posts Count

### PR DESCRIPTION
**Pull request by pasindud**
Posts count displayed using  {{{ posts_count }}}

### CODE REVIEW

**Comment by jacobmgreer** [comment]
!

File: lib/ruhoh/templaters/rmustache.rb [hunk 1]
```diff
@@ -55,6 +55,10 @@ def widget(name)
         Ruhoh::DB.widgets[name.to_s]['layout']
       end
       
+      def posts_count
```
Comment (plusjade) [inline lib/ruhoh/templaters/rmustache.rb:58]: No need to use a global variable here. The mustache class has direct access to the ruhoh database, so we can do something like:

``` ruby
def posts_count
  self.context['db']['posts']['dictionary'].length
end
```

**Comment by plusjade** [comment]
@pasindud thanks for contributing to ruhoh!  I've provided some feedback on your pull request. In general, global variables are usually a bad idea. 

Also, development on ruhoh 1.x will freeze as I ship out ruhoh 2.0. before this year is over. I'll make sure to provide a way to access the total post count though!

Thanks again and welcome to ruby!

**Comment by pasindud** [comment]
oh thanks 1st time on ruby

**Pull request closed without merging**